flink-tests/src/test/java/org/apache/flink/test/iterative/aggregators/AggregatorsITCase.java

```diff
@@ -170,12 +162,12 @@ public void testAggregatorWithParameterForIterate() throws Exception {
 				new NegativeElementsConvergenceCriterion());
 
 		DataSet<Integer> updatedDs = iteration.map(new SubtractOneMapWithParam());
-		iteration.closeWith(updatedDs).writeAsText(resultPath);
-		env.execute();
+		List<Integer> result = iteration.closeWith(updatedDs).collect();
+		Collections.sort(result);
```

Comment:
still some sort calls left.

Reply:
@zentol the result need be sorted, otherwise the test would report error, because the result is not ordered, print this : 

```
java.lang.AssertionError: 
Expected :[1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 5]
Actual   :[4, 2, 3, 5, 2, 5, 5, 5, 1, 3, 4, 4, 3, 5, 4]
```

Reply:
The correct answer would've been that we may sort the result since `TestBaseUtils#compareResultsByLinesInMemory` was also sorting both `expected` and `actual`.

Reply:
yes